packages/next/src/client/components/react-dev-overlay/_experimental/internal/helpers/merge-class-names.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Merge multiple args to a single string with spaces. Useful for merging class names.
+ *
+ * @example
+ * mergeClassNames('foo', 'bar') // 'foo bar'
+ * mergeClassNames('foo', null, 'bar', undefined, 'baz', false) // 'foo bar baz'
+ *
+ * @alias `cn()`
+ */
+export function mergeClassNames(
```

Comment:
we can use cx.ts, and only export `cx`, no need to have a long file name and duplicate exports